app/serializers/concerns/user_sidebar_mixin.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module UserSidebarMixin
+  def self.included(base)
+    base.attributes :display_sidebar_tags,
+                    :sidebar_tags,
+                    :sidebar_category_ids,
+                    :sidebar_list_destination
+  end
+
+  def sidebar_tags
+    object.visible_sidebar_tags(scope)
+      .pluck(:name, :topic_count, :pm_topic_count)
+      .reduce([]) do |tags, sidebar_tag|
+        tags.push(
+          name: sidebar_tag[0],
+          pm_only: sidebar_tag[1] == 0 && sidebar_tag[2] > 0
+        )
+      end
+  end
+
+  def include_sidebar_tags?
+    include_display_sidebar_tags?
+  end
```

Comment:
Most likely minor/theoretical, but should this check that there visible tags to include? In other words, should this check `display_sidebar_tags` instead of `include_display_sidebar_tags?`?